bin/www

```diff
@@ -40,12 +40,19 @@ if (key_file_location !== undefined && cert_file_location !== undefined) {
        key: fs.readFileSync(key_file_location),
        cert: fs.readFileSync(cert_file_location)
     };
-    server = https.createServer(options, app);
+    server = https.createServer(options, app).listen(https_port);
+
+    http.createServer(function (req, res) {
+        var url_location = "https://" + req.headers['host'].split(":")[0] + ":" + nconf.get('HTTPS_HOST_PORT') + req.url;
+        res.writeHead(301, { "Location": url_location});
+        res.end();
```

Comment:
I think it would be cleaner to rewrite above 2 lines as `res.redirect(301, url_location)`